AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/commands/core/PurgeCommand.java

```diff
@@ -0,0 +1,146 @@
+package io.github.niestrat99.advancedteleport.commands.core;
+
+import io.github.niestrat99.advancedteleport.CoreClass;
+import io.github.niestrat99.advancedteleport.commands.SubATCommand;
+import io.github.niestrat99.advancedteleport.config.CustomMessages;
+import io.github.niestrat99.advancedteleport.sql.HomeSQLManager;
+import io.github.niestrat99.advancedteleport.sql.SQLManager;
+import io.github.niestrat99.advancedteleport.sql.WarpSQLManager;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.util.StringUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class PurgeCommand implements SubATCommand {
+
+    // This command is going to purge warps and homes - the homes can be purged for the certain player or if the homes/warps are in a certain world.
+    // example: /at purge <warps|homes> <world|player> <World name|Player name>
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
+        // Check for if the arguments are lower than 3 - if that's the case the command will stop
+        if (args.length < 3) {
+            CustomMessages.sendMessage(sender, "Error.tooFewArguments");
+            return false;
+        }
+
+        // Check for if the arguments are not wrong, typo'd or missing - if that's the case the command will stop
+        if (!args[0].equalsIgnoreCase("warps") && !args[0].equalsIgnoreCase("homes")) {
+            CustomMessages.sendMessage(sender, "Error.invalidArgs");
+            return false;
+        }
+        if (!args[1].equalsIgnoreCase("world") && !args[1].equalsIgnoreCase("player")) {
+            CustomMessages.sendMessage(sender, "Error.invalidArgs");
+            return false;
+        }
+        if (args[1].equalsIgnoreCase("world")) {
```

Comment:
The condition from line 40-43 can be put at the end of the if-else statement (for args[2])